plugins/module_utils/vmware_rest.py

```diff
@@ -271,7 +270,7 @@ async def update_changed_flag(data, status, operation):
 
 
 async def list_devices(session, url):
-    existing_entries = []
+    pass
```

Comment:
this seems unnecessary, I would suggest removing this.  if we need to make changes to the generator to do that I'd be ok with merging this for now as it should be harmless and doing that under a separate PR.

Reply:
@jillr I don't think this file is generated. At least, when I generate code with ansible.content_builder (for example here: #441) I don't see it. But I may be wrong... I'm still learning and might have make a mistake somewhere.

Reply:
It might not be - I'm more familiar with the generated amazon collection and some of those module_utils lived in an early version of that generator, so calling it out just in case!

Reply:
@jillr It's already a very big PR with a lot of changes. I suggest to keep it like this and have a closer look at this later ;-)